release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -2165,7 +2165,7 @@ Issued: 2018-08-09
 
 {product-title} release 3.9.40 is now available. The list of packages and
 bug fixes included in the update are documented in the
-link:https://access.redhat.com/errata/RHSA-2018:2335[RHBA-2018:2335] advisory.
+link:https://access.redhat.com/errata/RHSA-2018:2335[RHSA-2018:2335] advisory.
```

Comment:
@tmorriso-rh The link should be:


    link:https://access.redhat.com/errata/RHBA-2018:2335[RHBA-2018:2335]

The RHSA path leads to a 404 error.